docs/modules/ROOT/pages/explanation/kubernetes.adoc

```diff
@@ -0,0 +1,162 @@
+By default, all INCOMING traffic must be allowed explicitly, while all OUTGOING traffic is allowed.
+
+Upon Namespace/Project creation, APPUiO Cloud will provision some default NetworkPolicies that ensure that
+
+- containers within the same namespace can communicate with each other
+- containers can be accessed from infrastructure components (monitoring, IngressController, etc.)
+
+[TIP]
+====
+See xref:how-to/remove-default-networkpolicies.adoc[Remove Default NetworkPolicies] on how to change this.
+====
+
+For most use cases this is enough, components that need to communicate directly tend to be deployed in the same namespace.
+In some cases however, we want components in different namespaces to be able to talk to each other.
+
+=== Example
+
+Let's assume our Application `myapp` consist of two parts, `frontend` and `api`.
+Each part
+
+- is deployed in its own namespace (`corp-myapp-frontend-prod` and `corp-myapp-api-prod` respectively)
+- consists of three pods
+- has a service pointing to it (named `frontend` and `api`)
+- is exposed via a Route/Ingress (using `myapp.example.com` and `api.example.com` as the URI)
+
+.Our example application
+image::explanation/network-policies.drawio.svg[]
+
+Let's also assume our frontend needs to talk to the API directly. To achieve this, we've 2 options:
+
+==== Use a Route/Ingress
+
+Since our API is already exposed to the world via an ingress, we can just configure our frontend to use `\https://api.example.com` as the API URL. This should work out of the box, no further configuration is needed.
+
+==== Create a Custom NetworkPolicy
+
+If our API isn't exposed to the internet directly, or we don't want our traffic to leave the cluster, we can create a NetworkPolicy that allows the frontend pods to talk to our API pods directly:
+
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-from-frontend  # <1>
+  namespace: corp-myapp-api-prod  # <2>
+spec:
+  policyTypes:
+    - Ingress
+  podSelector:
+    matchLabels:
+      app: api  # <3>
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: corp-myapp-frontend-prod  # <4>
+          podSelector:
+            matchLabels:
+              app: frontend  # <5>
+      ports:
+        - protocol: TCP
+          port: 3000  # <6>
+----
+<1> Give your policy a telling name, so you can identify what it's supposed to do later
+<2> The NetworkPolicy must be created in the DESTINATION namespace
+<3> Select which DESTINATION pods are affected by this NetworkPolicy
+<4> Select which SOURCE namespaces you want to allow traffic from. Labels must be used to select namespaces; the `kubernetes.io/metadata.name` label is added automatically to all namespaces
+<5> Select which specific SOURCE pods in the selected namespaces are allowed
+<6> Configure which ports traffic is allowed from
+
+[WARNING]
```

Comment:
```suggestion
[CAUTION]
```

See https://asciidoctor.org/docs/asciidoc-writers-guide/#admonitions.

Reply:
The guide you linked says

>Use WARNING to inform the reader of **danger, harm, or consequences** that exist.

There is the danger to expose your applications to untrusted third parties, which could cause harm or other consequences, so WARNING seems appropriate?

Reply:
… but the user must also **act carefully** …

Reply:
You are probably right. I was more looking at from a "proceed with caution to not mess up" perspective.